src/plugins/serverListIndicators/index.tsx

```diff
@@ -18,11 +18,14 @@
 
 import { addServerListElement, removeServerListElement, ServerListRenderPosition } from "@api/ServerList";
 import { Settings } from "@api/Settings";
+import { enableStyle } from "@api/Styles";
 import ErrorBoundary from "@components/ErrorBoundary";
 import { Devs } from "@utils/constants";
 import { useForceUpdater } from "@utils/react";
 import definePlugin, { OptionType } from "@utils/types";
-import { GuildStore, PresenceStore, RelationshipStore } from "@webpack/common";
+import { GuildStore, PresenceStore, RelationshipStore, Tooltip } from "@webpack/common";
+
+import style from "./styles.css?managed";
```

Comment:
there's no need for this to be managed! you only need to manually manage your styles when they change styling of stock discord ui
```suggestion
import "./styles.css";
```